src/sagemaker/jumpstart/curated_hub/accessors/curated_hub_s3_accessor.py

```diff
@@ -0,0 +1,102 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""This module accessors for the SageMaker JumpStart Curated Hub."""
+from __future__ import absolute_import
+from sagemaker.jumpstart.curated_hub.utils import (
+    get_model_framework,
+)
+from sagemaker.jumpstart.types import JumpStartModelSpecs
+from sagemaker.jumpstart.curated_hub.utils import (
+    get_studio_model_metadata_map_from_region,
+)
+from sagemaker.jumpstart.curated_hub.accessors.s3_object_reference import (
+    S3ObjectReference,
+    create_s3_object_reference_from_bucket_and_key,
+)
+from sagemaker.jumpstart.curated_hub.accessors.jumpstart_s3_accessor import JumpstartS3Accessor
+
+
+class CuratedHubS3Accessor(JumpstartS3Accessor):
+    """Helper class to access Curated Hub s3 bucket"""
+
+    def __init__(self, region: str, bucket: str):
+        self._region = region
+        self._bucket = bucket
+        self._studio_metadata_map = get_studio_model_metadata_map_from_region(
+            region
+        )  # Necessary for SDK - Studio metadata drift
+
+    def get_bucket(self) -> str:
+        """Retrieves s3 bucket"""
+        return self._bucket
+
+    def get_inference_artifact_s3_reference(
+        self, model_specs: JumpStartModelSpecs
+    ) -> S3ObjectReference:
+        """Retrieves s3 reference for model inference artifact"""
+        return create_s3_object_reference_from_bucket_and_key(
+            self.get_bucket(), f"{model_specs.model_id}/{model_specs.version}/infer.tar.gz"
+        )
```

Comment:
is this *always* the pattern of the inference s3 artifact? it's dangerous when things like this are hardcoded into the SDK, do we have to be careful.

Reply:
This is the format of where we will store the artifact in the HubBucket S3 - as long as it stays standardized here it will always be consistent